packages/swingset-liveslots/src/liveslots.js

```diff
@@ -553,15 +505,18 @@ function build(
   // use a slot from the corresponding allocateX
 
   function allocateExportID() {
-    return allocateNextID('exportID');
+    // eslint-disable-next-line no-use-before-define
+    return vrm.allocateNextID('exportID');
   }
 
   function allocateCollectionID() {
-    return allocateNextID('collectionID');
+    // eslint-disable-next-line no-use-before-define
+    return vrm.allocateNextID('collectionID');
   }
 
   function allocatePromiseID() {
-    const promiseID = allocateNextID('promiseID');
+    // eslint-disable-next-line no-use-before-define
+    const promiseID = vrm.allocateNextID('promiseID');
     return makeVatSlot('promise', true, promiseID);
```

Comment:
Don't we need to make corresponding changes to `tools/fakeVirtualSupport.js`?  Or are you expecting me to do that after this PR lands?

Reply:
OK, that means I did misunderstand (or, rather, forgot that particular detail).  I'll update `fakeVirtualSupport` to take out the faked ID management stuff.

Reply:
Do note that `packages/swingset-liveslots/test/virtual-objects/test-virtualObjectManager.js` will need some updates to golden tests to accommodate:

```js
    `get kindIDID => undefined`,
    `get idCounters => undefined`, // NEW: introduced by this PR
    `set kindIDID 1`,
```